executioncontext/projectview/src/test/resources/projectview/file7ImportsFile1.bazelproject

```diff
@@ -14,4 +14,6 @@ java_path: path7/to/java
 build_flags:
     --build_flag7.1=value7.1
     --build_flag7.2=value7.2
-    --build_flag7.3=value7.3
\ No newline at end of file
+    --build_flag7.3=value7.3
+
+build_manual_targets: false
```

Comment:
lets set it on true here - there will be a difference between file1 and this one